### PR TITLE
[MIG][10.0] sql_export

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Tasks done:
- Bump code to new version. Minor changes.

No extra features added/removed from v9

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
@florian-dacosta @jbeficent

File: sql_export/demo/sql_export.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@ Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
 License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 -->
 
-<openerp><data>
+<odoo><data>
```
Comment (yajo) [inline sql_export/demo/sql_export.xml]: Remove `<data>`, not required now.

File: sql_export/security/sql_export_security.xml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-<data noupdate="0">
+<odoo noupdate="0">
```
Comment (yajo) [inline sql_export/security/sql_export_security.xml]: `noupdate="0"` is the default, you can remove that, or use `noupdate="1"`, which is probably what the author wanted in the first place.

**Review by yajo — CHANGES_REQUESTED** [review]
code review

File: sql_export/README.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+SQL Export
+==========
+
+Allow to export data in csv files FROM sql requests.
+There are some restrictions in the sql sql request, you can only read datas.
```
Comment (florian-dacosta) [inline sql_export/README.rst]: in the sql sql request > in the sql query

**Review by florian-dacosta — APPROVED** [review]
Code review, no test
Just one small not blocking comment

Thanks for the port !

**Comment by AaronHForgeFlow** [comment]
@Yajo would you update your review?

**Review by yajo — APPROVED** [review]
code looks ok, but please squash all translation commits: 
captura de pantalla de 2017-11-30 12-33-28

I didn't do a functional test, I'd appreciate one from a real addon user (I don't use this one) to check migration works "Eficent"ly 😋

**Comment by Abranes** [comment]
Hello,
Tested but not work with parameters.
When I use any parameter to do a dynamic query I get an error:
`TypeError: 'dict' object does not support indexing`
This error come from https://github.com/OCA/server-tools/blob/86c6238433184f1a81f5a53a4c68306ff5697d79/sql_request_abstract/models/sql_request_mixin.py#L148
This function describe that the `params` need to be a `dict` but `mogrify` doesn't accept `dict` as a parameter.

**Comment by AaronHForgeFlow** [comment]
I will check

**Comment by AaronHForgeFlow** [comment]
Thank you @mreficent !

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
@Abranes could you check now if it works fine with paramenters?

**Pull request merged**